src/lib/client.cc

```diff
@@ -129,8 +129,7 @@ void
 Client::CreateBucket(const QString& name)
 {
 	ds3_request* request = ds3_init_put_bucket(name.toUtf8().constData());
-	LOG_INFO("Create Bucket " + name + " (PUT " + m_endpoint + "/" + \
-		 name + ")");
+	LOG_INFO("PUT          BUCKET    "+m_endpoint+"/"+name);
```

Comment:
Adding extra white space like this forces all callers to know how much whitespace to add so the logging looks right.  Instead, just log something like "PUT BUCKET system/bucketname".  If we really want to adjust the white space, to make the logs easier to read, then I'd rather the logging code handle that.